assets/js/colette-iframeResizer.js

```diff
@@ -0,0 +1,58 @@
+colette.iframeResizer = (function() {
+    'use strict';
+
+    var pub = {},
+        cfg = {
+            selector: '#iframeContent',
+            el: null,
+            delay: 0,
+            resizeInternal: false
+        };
+
+    pub.init = function(args)
+    {
+        cfg.delay = args.delay || cfg.delay;
+        cfg.selector = args.selector || cfg.selector;
+        cfg.el = document.querySelector(cfg.selector);
```

Comment:
Maybe no need to update ````selector```` and so:

    cfg.el = document.querySelector(args.selector || cfg.selector);